src/views/Home/AlertsView/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { Link } from 'react-router-dom';
+import {
+    gql,
+    useQuery,
+} from '@apollo/client';
+import { Container } from '@ifrc-go/ui';
+import { useTranslation } from '@ifrc-go/ui/hooks';
+import { _cs } from '@togglecorp/fujs';
+
+import {
+    CountryListQuery,
+    CountryListQueryVariables,
+} from '#generated/types/graphql';
+
+import AlertsAside from './AlertsAside';
+import AlertsMap from './AlertsMap';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+const COUNTRIES_LIST = gql`
+query CountryList {
+  public {
+    allCountries(alertFilters: {}) {
```

Comment:
```suggestion
    allCountries {
```

Can we not skip the filters altogether?